src/main/java/com/conveyal/r5/analyst/NetworkPreloader.java

```diff
@@ -140,7 +143,7 @@ protected TransportNetwork buildValue(Key key) {
                 linkedPointSet.getEgressCostTable(progressListener);
             }
         }
-        // Finished building all needed inputs for analysis, return the completed network to the AsyncLoader code.
```

Comment:
This method is also called by `synchronousPreload`, so I propose deleting this part of the comment.